modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -95,87 +82,553 @@ The following charts are available only if you are using Unified Agent and have
-* Inbound Response Time by Flow: Average, minimum, and maximum inbound response time
-* Outbound Response Time by Flow: Average, minimum, and maximum outbound response time
-* Inbound Request Volume by Flow: Average number of inbound requests
-* Outbound Request Volume by Flow: Average number of outbound requests
-* Failures by Flow: Average number of inbound failed requests
-* Slow Inbound Requests by Flow: Average number of inbound requests whose response time is greater than 1000 ms
-* Slow Outbound Requests by Flow: Average number of outbound requests whose response time is greater than 1000 ms
+| <<flows-charts>> a|
+
+* Inbound Response Time by Flow
+* Outbound Response Time by Flow
+* Inbound Request by Flow
+* Outbound Request by Flow
+* Failures by Flow
+* Slow Inbound Requests by Flow
+* Slow Outbound Requests by Flow
+* Inbound Response Time by Flow
+* Outbound Response Time by Flow
+* Inbound Request Volume by Flow
+* Outbound Request Volume by Flow
+* Failures by Flow
+* Slow Inbound Requests by Flow
+* Slow Outbound Requests by Flow
 
 If you don't see the *Flows* tab, verify that you are using Unified Agent and have a Titanium subscription to Anypoint Platform.
-| *Connectors* a|
-
-* Connector Message Volume: Number of outbound messages and number of outbound requests
-* Response Time by Connector: Average outbound response time
+| <<connectors-charts>> a|
+
+Connector Message Volume
+
+* Response Time by Connector
+* Salesforce Requests
+* Salesforce Response Time
+* Salesforce Failures
+* Database Requests
+* Database Response Time
+* Database Failures
+* HTTP Requests
+* HTTP Response Time
+* HTTP Failures
+* Web Services Requests
+* Web Services Response Time
+* Web Services Failures
+* FTP Requests
+* FTP Response Time
+* FTP Failures
+* SFTP Requests
+* SFTP Response Time
+* SFTP Failures
+* SMTP Requests
+* SMTP Response Time
+* SMTP Failures
+* Object Store Requests
+* Object Store Response Time
+* Object Store Failures
+* Anypoint MQ Operations - Inbound
+* Anypoint MQ Response Time - Inbound
+* Anypoint MQ Failures - Inbound
+* Anypoint MQ Operations - Outbound
+* Anypoint MQ Response Time - Outbound
+* Anypoint MQ Failures - Outbound
+* Other Connector Requests
+* Other Connector Response Time
+* Other Connector Failures
 
 If you don't see the *Connectors* tab, verify that you have a Titanium subscription to Anypoint Platform and that the app you selected is a hybrid app deployed to a server, server group, or cluster. See xref:monitor-connectors.adoc[Monitor Connectors].
-| *Performance* a|
+| <<performance-charts>>* a|
 
 * Average Response Time - Inbound
 * Average Response Time Grouped by Endpoint - Inbound
 * Mule Messages
-* Response Time graphs: 99, 90, 75, and 50 percentile inbound
+* Response Time graphs (99, 90, 75, and 50 percentile) - Inbound
 * Average Response Time  - Outbound
 * Average Response Time Grouped by Endpoint - Outbound
```

Comment:
Also missing "Outbound - Response Time by HTTP Endpoint", a table with endpoint and Avg/Min/Max